templates/artist/artist_detail.html

```diff
@@ -47,35 +47,19 @@
 {% endblock %}
 
 {% block content %}
-    <h1 class="header-font">{{ artist.name }}</h1>
-    {% if investors or artist.past_campaigns or updates or has_permission_to_submit_update %}
```

Comment:
Ahh... my goal here was to show all of the tabs so that investors / potential investors could see there was a section for "updates" or "past campaigns" to know those were things that could happen in the future. Right now most people don't know they can get updates from the artist unless they are invested and the artist has posted one.